README.md

```diff
@@ -50,10 +52,47 @@ In addition to `live_view_native` you may want to include some additional librar
 {:live_view_native_live_form, "~> 0.3.0-rc.3"}
 ```
 
+The versions above may be subject to change, so make sure to check the library documentation for the latest version.
+* [LiveView Native Versions](https://hex.pm/packages/live_view_native/versions)
+* [LiveView Native Stylesheet Versions](https://hex.pm/packages/live_view_native_stylesheet/versions)
+* [LiveView Native SwiftUI Client Versions](https://hex.pm/packages/live_view_native_swiftui/versions)
+* [LiveView Native Live Form Versions](https://hex.pm/packages/live_view_native_live_form/versions)
+
 Then run:
 
 ```
 $ mix lvn.setup
 ```
 
 and follow the instructions on how to complete the setup process.
+
+## Main Branch
+
+To try out new features or get the latest bugfixes you can also use the `main` branch for LiveView Native dependencies.
+
+```elixir
+{:live_view_native, github: "liveview-native/live_view_native", override: true},
+{:live_view_native_stylesheet, github: "liveview-native/live_view_native_stylesheet"},
+{:live_view_native_swiftui, github: "liveview-native/liveview-client-swiftui"},
+{:live_view_native_live_form, github: "liveview-native/liveview-native-live-form"}
+```
+
+However, be aware that active changes may make your development experience unstable.
+
+## Native Clients
+
+LiveView Native enables native client frameworks such as the [LiveView Native SwiftUI Client](https://github.com/liveview-native/liveview-client-swiftui) and the [LiveView Native Jetpack Client (In Progress)](https://github.com/liveview-native/liveview-client-jetpack).
```

Comment:
I think we should have a matrix of clients that we're listing include the html client

Reply:
That sounds good to me, but I'm not sure how you want that to look as a matrix 🤔

Reply:
Just a table maybe?

| UI Framework | Devices, | LiveView Client | 
|---------------|----------|----------------|
| SwiftUI            | iPhone, iPad, AppleTV, Apple Watch, MacOS, Apple Vision Pro | [LiveView Native SwiftUI](https://github.com/liveview-native/liveview-client-swiftui) |
| JetPack Compose | Android family | [LiveView Native Jetpack](https://github.com/liveview-native/liveview-client-jetpack) |

Reply:
I made it a list of all of them. Let me know if that's not what you have in mind and how to improve it

Reply:
See the table above, let's go with that. You should be able to copy/paste that in and just add a HTML one as well

Reply:
@BrooklinJazz ☝️ can you change to the table form?

Reply:
Will do, sorry there was a desync issue when I asked the question so I hadn't seen the table

Reply:
Done